.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
```

Comment:
Should this be `kind:bug` according to the contributor guide? Also, we should probably just not talk about labels in the contributor file since people opening issues usually don't have permissions to assign labels themselves.